website/docs/d/alert_configuration.html.markdown

```diff
@@ -226,6 +226,7 @@ Notifications to send when an alert condition is detected.
     - `WEBHOOK`
     - `MICROSOFT_TEAMS`
 
+* `notifier_id` - The notifier id is a system-generated unique identifier assigned to each notification method. This is needed when updating third-party notifications without requiring explicit authentication credentials.
```

Comment:
Do you need to update https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/alert_configurations as well?

Reply:
thanks, adjusted. Plural data source had missing docs for matchers, threshold config, metric threshold config, and notification attributes.